app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :posts
+  validates :name, presence: true
+  validates :username, presence: true
+  validates :password, presence: true
+  validates :email, presence: true
+  validates :password_confirmation, presence: true
```

Comment:
- [ ] [OPTIONAL] Keep your code DRY at all times. You could write the validations in a single line since they all have the same property. `validates :name, :username, :password, :email, presence: true`